src/transformers/generation/utils.py

```diff
@@ -2318,15 +2362,17 @@ def greedy_search(
             if synced_gpus and this_peer_finished:
                 continue  # don't waste resources running the code we don't need
 
-            next_token_logits = outputs.logits[:, -1, :]
+            next_tokens_logits = outputs.logits[:, -1, :]
```

Comment:
```suggestion
            next_token_logits = outputs.logits[:, -1, :]
```
logits (plural) regarding the next token (singular)